tests/base/qprojector.output

```diff
@@ -1925,47 +1925,47 @@ DEAL:face::0.75 1.0 1.0
 DEAL:face::1.0 1.0 1.0
 DEAL:all::Checking dim 2 1d-points 5
 DEAL:all::Face 0 orientation false
-DEAL:all::0.0 0.0
-DEAL:all::0.0 0.25
-DEAL:all::0.0 0.50
-DEAL:all::0.0 0.75
 DEAL:all::0.0 1.0
+DEAL:all::0.0 0.75
+DEAL:all::0.0 0.50
+DEAL:all::0.0 0.25
+DEAL:all::0.0 0.0
```

Comment:
Same here? If I see this right, the points shown are exactly the same as for the face orientation = true case

Reply:
Yup: this test uses
```cpp